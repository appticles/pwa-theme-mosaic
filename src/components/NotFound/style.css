.ui.container + div {
  display: none;
}

.ui.container {
  height: unset;
}

.ui.container h1.ui.header {
  color: #000000;
  position: static;
}
