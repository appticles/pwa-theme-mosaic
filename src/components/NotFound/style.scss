@import "../../utilities/variables";

.not-found.ui.header {
  display: flex;
  align-items: center;
  padding-top: $base-spacing-xl;
}
