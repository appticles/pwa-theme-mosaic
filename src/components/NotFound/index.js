import React from 'react';
import { Header, Container } from 'semantic-ui-react';
import PropTypes from 'prop-types';

import './style.css';

const NotFound = props => (
  <Container className="not-found">
    <Header> {props.content}</Header>
  </Container>
);

NotFound.defaultProps = {
  content: 'Sorry, we could not find what you were looking for.',
};
NotFound.propTypes = {
  content: PropTypes.string,
};
export default NotFound;
