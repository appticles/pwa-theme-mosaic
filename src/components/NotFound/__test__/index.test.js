import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import NotFound from '../index';

describe('Not found component', () => {
  it('should render correctly', () => {
    const output = shallow(<NotFound />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
