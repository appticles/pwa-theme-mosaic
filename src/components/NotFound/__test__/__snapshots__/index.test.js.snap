// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found component should render correctly 1`] = `
<Header
  as="h3"
  className="not-found"
  icon={true}
  textAlign="center"
>
  <Icon
    as="i"
    circular={true}
    name="folder open"
  />
  <HeaderContent>
    Sorry, we could not find what you were looking for.
  </HeaderContent>
</Header>
`;
